docs/vm-installation.md

```diff
@@ -0,0 +1,115 @@
+kubectl config --kubeconfig=antrea-agent.antrea.kubeconfig set-cluster antrea --server=$ANTREA_API_SERVER --insecure-skip-tls-verify=true
+kubectl config --kubeconfig=antrea-agent.antrea.kubeconfig set-credentials antrea-agent --token=$TOKEN
+kubectl config --kubeconfig=antrea-agent.antrea.kubeconfig set-context antrea-agent@antrea --cluster=antrea --user=antrea-agent
+kubectl config --kubeconfig=antrea-agent.antrea.kubeconfig use-context antrea-agent@antrea
+```
+
+
+## Prerequisites on Linux VM
+
+OVS needs to be installed. Please refer to the [getting-started guide](getting-started.md#open-vswitch)
+
+## Running antrea-agent on Linux VM
+
+1. Generate `antrea-agent` binaries
+
+```bash
+make docker-bin
+```
+
+2. Create `antrea-agent` service: Below is a sample snippet to start `antrea-agent` as a service on Ubuntu
+
+```bash
+mkdir -p /var/log/antrea
+cat << EOF > /etc/systemd/system/antrea-agent.service
+Description="antrea-agent as a systemd service"
+After=network.target
+[Service]
+ExecStart=/usr/sbin/antrea-agent \
+--config=/root/antrea/antrea-agent.conf \
+--logtostderr=false \
+--log_file=/var/log/antrea/antrea-agent.log
+--v=4
+Restart=on-failure
+[Install]
+WantedBy=multi-user.target
+EOF
+
+sudo systemctl daemon-reload
+sudo systemctl enable antrea-agent
+sudo systemctl start antrea-agent
+```
+
+3. The `antrea-agent` configuration file specifies agent configuration parameters. For all the agent configuration
+   parameters of VM, refer to this [base configuration file](../build/yamls/externalnode/conf/antrea-agent.conf).
+   Update `clientConnection` and `antreaClientConnection` with the corresponding kubeconfig file path.
+
+## Prerequisites on Windows VM
+
+1. OVS needs to be installed. For more information refer to the [Antrea Windows documentation](windows.md#1-optional-install-ovs-provided-by-antrea-or-your-own).
+2. Enable Hyper-V optional feature on Windows VM.
+
+```powershell
+Install-WindowsFeature Hyper-V-Powershell
+Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Hyper-V -All -NoRestart
+```
+
+3. Download [nssm](https://nssm.cc/download) to configure antrea-agent as a service.
+
+## Running antrea-agent on Windows VM
+
+1. Generate `antrea-agent` binaries
+
+```powershell
+make docker-windows-bin
+```
+
+2. Create `antrea-agent` service using nssm: Below is a sample snippet to start `antrea-agent` as a service
+
+```powershell
+mkdir C:\antrea-agent
+nssm.exe install antrea-agent --config C:\antrea-agent\antrea-agent.conf --log_file C:\antrea-agent\antrea-agent.log --logtostderr=false
+nssm.exe start antrea-agent
+```
+
+3. The `antrea-agent` configuration file specifies agent configuration parameters. For all the agent configuration
+   parameters of VM, refer to this [base configuration file](../build/yamls/externalnode/conf/antrea-agent.conf).
```

Comment:
"Please refer to this base configuration file to check all configuration parameters for VMs"?